src/main/java/mate/academy/spring/config/SecurityConfig.java

```diff
@@ -28,15 +31,16 @@ protected void configure(HttpSecurity http) throws Exception {
         http.authorizeRequests()
                 .antMatchers("/register").permitAll()
                 .antMatchers(HttpMethod.GET, "/cinema-halls", "/movies",
-                        "/movie-sessions/available").hasAnyRole("ADMIN", "USER")
+                        "/movie-sessions/available").hasAnyRole(ADMIN_ROLE, USER_ROLE)
                 .antMatchers(HttpMethod.POST, "/cinema-halls",
-                        "/movies", "/movie-sessions").hasRole("ADMIN")
-                .antMatchers(HttpMethod.GET, "/users/by-email").hasRole("ADMIN")
-                .antMatchers(HttpMethod.PUT, "/movie-sessions/{id}").hasRole("ADMIN")
-                .antMatchers(HttpMethod.DELETE, "/movie-sessions/{id}").hasRole("ADMIN")
-                .antMatchers(HttpMethod.POST, "/orders/complete").hasRole("USER")
-                .antMatchers(HttpMethod.GET, "/orders", "/shopping-carts/by-user").hasRole("USER")
-                .antMatchers(HttpMethod.PUT, "/movie-sessions").hasRole("USER")
+                        "/movies", "/movie-sessions").hasRole(ADMIN_ROLE)
+                .antMatchers(HttpMethod.GET, "/users/by-email").hasRole(ADMIN_ROLE)
+                .antMatchers(HttpMethod.PUT, "/movie-sessions/{id}").hasRole(ADMIN_ROLE)
+                .antMatchers(HttpMethod.DELETE, "/movie-sessions/{id}").hasRole(ADMIN_ROLE)
```

Comment:
ok, but it would be better to write `ADMIN.name()` directly here, without constants
we can change import if we don't won't to write `Role.RoleName.`